ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java

```diff
@@ -712,7 +712,7 @@ public synchronized void measureInWindow(int tag, int[] outputBuffer) {
     Rect visibleWindowFrame = new Rect();
     v.getWindowVisibleDisplayFrame(visibleWindowFrame);
     outputBuffer[0] = outputBuffer[0] - visibleWindowFrame.left;
-    outputBuffer[1] = outputBuffer[1] - visibleWindowFrame.top;
+    outputBuffer[1] = outputBuffer[1] + visibleWindowFrame.top;
```

Comment:
From discussion with @yungsters, why are we adding visibleWindowFrame.top versus removing this line altogether?

Also from the PR that introduced this change: https://github.com/facebook/react-native/pull/28449
visibleWindowFrame.top might include more than just status bar height.. is that expected that we want this included in the measure values? 

Has this change been tested in split screen?

Reply:
Hi @lunaleaps, let me check again. This PR is more than 1 year old already and I forget about the context. Let me check and confirm your concern and test the behavior on split-screen.

Reply:
Hi @lunaleaps i will work on this on the weekend, sorry for the delay.

Reply:
No worries, thanks for working on this!